.github/workflows/test.yml

```diff
@@ -44,3 +47,58 @@ jobs:
           source activate          
           pip install pytest
           pytest
+
+  paths-filter:
+    runs-on: ubuntu-latest
+    outputs:
+      tag-file-modified: ${{ steps.filter.outputs.tag-file }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Path Filter
+        id: filter
+        uses: dorny/paths-filter@v3
+        with:
+          filters: |
+            tag-file:
+              - 'ersilia/hub/content/metadata/tag.txt'
+
+      - name: Tag file modified
+        if: steps.filter.outputs.tag-file == 'true'
+        run: echo "Tag file modified"
+
+      - name: Tag file not modified
+        if: steps.filter.outputs.tag-file != 'true'
+        run: echo "Tag file not modified"
+
+  update-model-request:
```

Comment:
Please rename to "update-model-request-template"

Reply:
done